addons/web/static/src/views/model.js

```diff
@@ -98,6 +98,26 @@ export function useModel(ModelClass, params, options = {}) {
     if (!("isAlive" in params)) {
         params.isAlive = () => status(component) !== "destroyed";
     }
+    if (services.dialog) {
```

Comment:
nitpicking: could you move this above (line 97), in the block related to services?

Reply:
done